odbc/src/lib.rs

```diff
@@ -1,3 +1,6 @@
+pub mod api;
```

Comment:
I can't think of any reason for this to be `pub`
```suggestion
mod api;
```

Reply:
how would this work if it's not pub? I'm pretty sure it won't build the symbols

Reply:
Oh right, `pub use`. I'm silly.